coalib/bearlib/abstractions/Linter.py

```diff
@@ -1,5 +1,6 @@
 from contextlib import contextmanager
 from functools import partial, partialmethod
```

Comment:
If you want you can also copy the commit body from the other commit to explain why this change is made :+1: